src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketOptionName.cs

```diff
@@ -137,6 +137,7 @@ public enum SocketOptionName
         BsdUrgent = 2,
         Expedited = 2,
         TcpKeepAliveRetryCount = 16,
+        FastOpen = 15,
```

Comment:
We should document the concerns (=fully controlled datapath needed) and the limitations around this flag.

Reply:
yes, I have no problem with good description. Even the man page for Linux talks about differences based on cookie availability.